tsconfig.json

```diff
@@ -11,5 +11,15 @@
         "strictNullChecks": false,
         "noUnusedLocals": true
     },
-    "include": ["src/**/*.ts", "src/**/*.tsx", ".eslintrc.js", "src/**/*.js"]
+    "include": [
+        ".eslintrc.js",
```

Comment:
This is redundant with `"./*.js"`, no?

Reply:
It is. But if I don't include it, I get an error parsing .eslintrc.js. My theory is the beginning '.' messes up the glob logic.